app-services/secrets/main.go

```diff
@@ -73,38 +73,77 @@ func main() {
 	os.Exit(0)
 }
 
-type getSecretsTest struct {
```

Comment:
I am confused why the tests are in `main.go` rather than `main_test.go`

Reply:
fixed it